### PR TITLE
Update Elasticsearch Docker compose doc

### PR DESCRIPTION
**Pull request by Skrypt**
Update Docker compose examples for devs.

This is instead of https://github.com/OrchardCMS/OrchardCore/pull/17167.

### CODE REVIEW

File: src/docs/reference/modules/Elasticsearch/Elasticsearch-8.x/.env [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Password for the 'elastic' user (at least 6 characters)
+ELASTIC_PASSWORD=password
```
Comment (Piedone) [inline src/docs/reference/modules/Elasticsearch/Elasticsearch-8.x/.env:2]: The appsettings sample config needs to be changed too.

**Review by Piedone — CHANGES_REQUESTED** [review]
Also update the docs along the lines of: https://github.com/OrchardCMS/OrchardCore/pull/17167/files#diff-2929d3134dcabd269bbfc32ffee089859fd6f733d587054e9d0e6f3b9b8933cc

File: src/docs/reference/modules/Elasticsearch/Elasticsearch-8.x/.env [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# Password for the 'kibana_system' user (at least 6 characters)
+KIBANA_PASSWORD=password
+
+# Version of Elastic products
+STACK_VERSION=8.16.1
+
+# Set the cluster name
+CLUSTER_NAME=docker-cluster
+
+# Set to 'basic' or 'trial' to automatically start the 30-day trial
```
[the remark this answers is not in the document]
Comment (Piedone) [reply src/docs/reference/modules/Elasticsearch/Elasticsearch-8.x/.env:16]: OK then!

File: src/docs/reference/modules/Elasticsearch/README.md [hunk 3]
```diff
@@ -28,11 +28,10 @@ Then exit any WSL instance, `wsl --shutdown`, and restart.
 vm.max_map_count = 262144
 ```
 
-Elasticsearch v7.17.5 Docker Compose file :
-[docker-compose.yml](docker-compose.yml)
+Elasticsearch Docker v7.x Compose file :
```
Comment (Piedone) [inline src/docs/reference/modules/Elasticsearch/README.md:31]:
```suggestion
Elasticsearch Docker v7.x Compose file:
```

File: src/docs/reference/modules/Elasticsearch/README.md [hunk 4]
```diff
@@ -41,6 +40,21 @@ You should get this result in Docker Desktop app:
 
 ![Elasticsearch docker containers](images/elasticsearch-docker.png)
 
+Elasticsearch Docker v8.x Compose file :
```
Comment (Piedone) [inline src/docs/reference/modules/Elasticsearch/README.md:43]:
```suggestion
Elasticsearch Docker v8.x Compose file:
```

File: src/docs/reference/modules/Elasticsearch/README.md [hunk 5]
```diff
@@ -28,11 +28,10 @@ Then exit any WSL instance, `wsl --shutdown`, and restart.
...
 ```
 
-Elasticsearch v7.17.5 Docker Compose file :
-[docker-compose.yml](docker-compose.yml)
+Elasticsearch Docker v7.x Compose file :
+[docker-compose.yml](Elasticsearch-7.x/docker-compose.yml)
 
-- Copy this file in a folder named Elasticsearch somewhere safe.
-- Open up a Terminal or Command Shell in this folder.
+- Open up a Terminal or Command Shell in the Elasticsearch-7.x folder.
```
Comment (Piedone) [inline src/docs/reference/modules/Elasticsearch/README.md:34]:
```suggestion
- Open up a Terminal or Command Shell in the `Elasticsearch-7.x` folder.
```

File: src/docs/reference/modules/Elasticsearch/README.md [hunk 6]
```diff
@@ -41,6 +40,21 @@ You should get this result in Docker Desktop app:
 
 ![Elasticsearch docker containers](images/elasticsearch-docker.png)
 
+Elasticsearch Docker v8.x Compose file :
+[docker-compose.yml](Elasticsearch-8.x/docker-compose.yml)
+
+- Open up a Terminal or Command Shell in the Elasticsearch-8.x folder.
```
Comment (Piedone) [inline src/docs/reference/modules/Elasticsearch/README.md:46]:
```suggestion
- Open up a Terminal or Command Shell in the `Elasticsearch-8.x` folder.
```

**Review by Piedone — COMMENTED** [review]
Let me know when you're again ready for a review (note the open comments).

File: src/docs/reference/modules/Elasticsearch/Elasticsearch-8.x/.env (hunk 1 above)
Comment (Skrypt) [reply src/docs/reference/modules/Elasticsearch/Elasticsearch-8.x/.env]: Yeah I'm trying to make it work with the elastic/changeme creds and it doesn't create any index yet. Seems like it is something that doesn't work anymore or I'm missing a config in the appsettings.json file.

Comment (Piedone) [reply src/docs/reference/modules/Elasticsearch/Elasticsearch-8.x/.env]: OK, thanks.

Comment (Skrypt) [reply src/docs/reference/modules/Elasticsearch/Elasticsearch-8.x/.env]: Ok, so I think the issue is that I need to get a Fingerprint from Elasticsearch. I'm trying to find how to get it. Doesn't seem so obvious. Not sure if it is worth it at this point. Maybe better disable auth but that's a shame.

Comment (Skrypt) [reply src/docs/reference/modules/Elasticsearch/Elasticsearch-8.x/.env]: Ok mets merge your PR. I will keep this one here if I ever find thé trick with auth.

Comment (Piedone) [reply src/docs/reference/modules/Elasticsearch/Elasticsearch-8.x/.env]: OK then!

**Comment by Piedone** [comment]
Please reopen then, if necessary.

**Pull request closed without merging**